webextensions/api/windows.json

```
@@ -667,9 +667,9 @@
               "firefox": {
                 "version_added": "45",
                 "notes": [
-                  "'url' and 'tabId options can't both be set together.",
-                  "The returned 'Window' object contains the 'tabs' property only from version 52 onwards.",
-                  "From Firefox 86, the <code>focused: false</code> option is ignored.",
+                  "<code>url</code> and <code>tabId</code> options can't be set together.",
+                  "The returned <code>Window</code> object contains the <code>tabs</code> property only from version 52 onwards.",
+                  "In Firefox 85 and earlier, the <code>focused: false</code> is unsupported. From Firefox 86, <code>focused: false</code> is ignored",
```

typo. What is the difference between unsupported and ignored?

```suggestion
                  "In Firefox 85 and earlier, <code>focused: false</code> is unsupported. From Firefox 86, <code>focused: false</code> is ignored.",
```

---

Clarified - no supported means an error if you use it.
```suggestion
                  "In Firefox 85 and earlier, <code>focused: false</code> is unsupported and throws an error. From Firefox 86, <code>focused: false</code> is ignored.",```